addons/hr_timesheet_attendance/tests/test_timesheet_attendance.py

```diff
@@ -0,0 +1,29 @@
+# -*- coding: utf-8 -*-
+
+from datetime import datetime
+
+from odoo.addons.hr_timesheet.tests.test_timesheet import TestCommonTimesheet
+
+class TestTimesheetAttendance(TestCommonTimesheet):
```

Comment:
We could perhaps launch this unit test in post_install to be sure the behavior remains the same.

What do you think? 🙂